app/views/files/_form.html.haml

```diff
@@ -7,7 +7,10 @@
   = form.form_group t('.current_file') do
     - if @file.is_image?
       .file-image= image_tag @file.file.url
-    = link_to @file.file.url, @file.file.url, target: '_blank'
+    %ul
+      - FilesHelper::IMAGE_STYLES.each do |size|
+        %li
+          = link_to size.humanize, @file.file.url(size.to_sym), target: '_blank'
```

Comment:
@tomas-stefano there's fix for this issue [here](https://github.com/moneyadviceservice/comfortable-mexican-sofa) Wdyt?